app/controllers/vm_cloud_controller.rb

```diff
@@ -113,16 +113,12 @@ def live_migrate
   def live_migrate_form_fields
     assert_privileges("instance_live_migrate")
     @record = find_by_id_filtered(VmOrTemplate, params[:id])
-    clusters = []
-    hosts = []
-    @record.ext_management_system.find_filtered_children("ems_clusters").each do |c|
-      clusters << {:id => c.id, :name => c.name}
+    clusters = @record.ext_management_system.ems_clusters.map do |c|
+      {:id => c.id, :name => c.name}
     end
-    @record.ext_management_system.find_filtered_children("hosts").each do |h|
-      hosts << {:id => h.id, :name => h.name, :cluster_id => h.emd_cluster.id}
+    hosts = @record.ext_management_system.hosts.map do |h|
```

Comment:
`find_filtered_children` assumed you are passing in a filter. We never were, so that code was pretty much useless.

If you're trying to filter the results based upon rbac, then maybe we need to introduce a method of easily filtering this data.